### PR TITLE
add WT32_ETH01

### PR DESCRIPTION
**Pull request by ermaya**

### CODE REVIEW

**Review by przemyslawzygmunt — APPROVED** [review]

File: src/supla/network/wt32_eth01.h [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+#endif
+      long sendSize = client->write((const uint8_t *)buf, count);
+      return sendSize;
+    }
+
+    int connect(const char *server, int port = -1) {
+      String message;
+      if (client != NULL) {
+        client = NULL;
+      }
```
Comment (klew) [inline src/supla/network/wt32_eth01.h]: you are creating client instance with "new" operator, but there is no delete for it. So this code will leak memory on each attempt to connect

File: src/supla/network/wt32_eth01.h [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#ifndef wt32_eth1_h__
```
Comment (klew) [inline src/supla/network/wt32_eth01.h:18]: we start to have more and more "network interfaces" and people may get lost. Could you please ass in commet short infor for what boards and what ETH hardware this network interface can be used? Please add similar explanation in the other ETH file

**Review by klew — CHANGES_REQUESTED** [review]

File: src/supla/network/wt32_eth01.h (hunk 2 above)
Comment (ermaya) [reply src/supla/network/wt32_eth01.h:18]: added

File: src/supla/network/wt32_eth01.h (hunk 1 above)
Comment (ermaya) [reply src/supla/network/wt32_eth01.h]: done

**Review by klew — APPROVED** [review]

**Pull request merged**